rip/utils.py

```diff
@@ -1,8 +1,30 @@
 import openai
+import re
 from api_secrets import API_KEY, ORGANIZATION
 
+DEFAULT_IMG_URL = "https://oaidalleapiprodscus.blob.core.windows.net/private/org-WUj2RT1EDW0dxp5pdZ1cixJz/user-VyFZAHxqEZ2X6M0XJ98C2zbc/img-XmAd24nVMBaHPeBoPzDNIen7.png?st=2022-11-12T13%3A55%3A31Z&se=2022-11-12T15%3A55%3A31Z&sp=r&sv=2021-08-06&sr=b&rscd=inline&rsct=image/png&skoid=6aaadede-4fb3-4698-a8f6-684d7786b067&sktid=a48cca56-e6da-484e-a814-9c849652bcb3&skt=2022-11-11T23%3A13%3A21Z&ske=2022-11-12T23%3A13%3A21Z&sks=b&skv=2021-08-06&sig=95hXSEujfXMDZ204VRhQTTQdv6tuc9e2L1ICzuYGQd8%3D"
+
```

Comment:
I get AuthenticationFailed error, maybe we can start with empty placeholder 🤔 

<Error>
<Code>AuthenticationFailed</Code>
<Message>Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature. RequestId:7f1b75ee-001e-001f-6a35-f7a1c6000000 Time:2022-11-13T07:57:13.4549845Z</Message>
<AuthenticationErrorDetail>Signature not valid in the specified time frame: Start [Sat, 12 Nov 2022 13:55:31 GMT] - Expiry [Sat, 12 Nov 2022 15:55:31 GMT] - Current [Sun, 13 Nov 2022 07:57:13 GMT]</AuthenticationErrorDetail>
</Error>

Reply:
agreed, or I added an image in our git repo so that we can use it as a default image. And this new committed version is using that URL :)